### ISSUE TITLE
HTML Exercise 1 and 2 Feedback

### ISSUE THREAD

**Issue by RJP43** [JaredKramer40/personalWebpage#1]
@JaredKramer40 I am not seeing your submission for this assignment here in your GitHub repo or at your PSC address (http://www.newtfire.org/~jlk186/). In fact, @ebeshero I do not see Jared's folder in the home directory of the PSC and I wonder if I am just missing his folder or if it even exists. @JaredKramer40 did you complete [HTML Exercise 1](http://newtfire.org/dh/HTMLExercise1.html)? If this is an issue with PSC access please at least get your files up here on GitHub so we can comment on them and you can make adjustments as per our suggestions for [HTML Exercise 2](http://newtfire.org/dh/HTMLExercise2.html).

**Comment by JaredKramer40** [comment]
@RJP43 Yeah I am having troubles with my FTP client.  I completed it but can't get it up.  I will put it up here.

**Comment by RJP43** [comment]
Great thanks, @JaredKramer40 !

Here are some preliminary comments regarding your [webpage](http://www.newtfire.org/~jlk186/) on newtfire.org:

I think there is a bigger issue here with your PSC access then you could fix on your own and you should meet with @ebeshero and me after class tomorrow to sort this out. Remember as you do [HTML Exercise 2](http://newtfire.org/dh/HTMLExercise2.html) to move ALL files from your local Git workspace to GitHub. It looks like you link to an image on your index.html page (JaredKramer1.jpg), but since you have not included it in this repo the link is broken when I look at your site using the [GitHub prefix](http://htmlpreview.github.io/?https://github.com/JaredKramer40/personalWebpage/blob/master/index.html) to view your site.  

### Here are my suggestions to better your webpage:

You did a nice job correctly linking your CSS to your HTML page, well done! And I like that you experimented with a variety of HTML elements (including `<ul>`, `<p>`, and different size heading elements). 

For [HTML Exercise 2](http://newtfire.org/dh/HTMLExercise2.html) we would like to see you make at least one interior link to a subpage of this main page (perhaps a photo collection or resume page) as well as some more external links (linking out to other sites).  Try experimenting by sectioning your different content pieces in separate `<div>` elements like you have with your image. `<div>` elements create clear distinctions in content sections and help with grouped content styling. Perhaps see what your image would look like beside your text. 

As for your CSS styling you have done well keeping things simple, but as it becomes more complicated consider using comment tags (`/*comment in here*/`) to organize the different sections of styling that go with different pages. 

Be aware that your background and `<h1>` color choice clash, which makes it difficult to read. It is important to always consider how your text looks when you choose very distinct colors. You might consider adding a neutral colored `<div>` that the text portions sit inside or mess around with different fonts, font sizes, font thickness for different parts of the content (ie. headings, paragraphs) that might make the text easier to read on the colored background.

Great first start and we will hopefully get you running on the PSC tomorrow!

Check CourseWeb for your grade on this assignment. @ebeshero and I look forward to seeing how you change your page for Exercise 2!

**Comment by RJP43** [comment]
@JaredKramer40 ahh I see you have been making edits as I was typing this out! Looking better already!

**Comment by JaredKramer40** [comment]
Thanks! I work at 11:30 after class tomorrow is there any other time you can help me with that?

**Comment by RJP43** [comment]
I can come in earlier as in before class. @ebeshero I think this is an issue I need your help to resolve. Could you come in a little earlier tomorrow as well?

**Comment by RJP43** [comment]
Also, @JaredKramer40 toss this prefix (http://htmlpreview.github.io/?) in front of any of your HTML pages here on GitHub and you can see how your site looks. 

Here is your index page: http://htmlpreview.github.io/?https://github.com/JaredKramer40/personalWebpage/blob/master/index.html

You won't be able to see the SSI (which we ask you to create in HTML 2)  using this prefix, but I can let you know if your SSI will work if you include the link and file in the repo.  Read through the[ SSI tutorial](http://newtfire.org/dh/SSI.html) to get that set up.

**Comment by RJP43** [comment]
@JaredKramer40 does tomorrow before class work for you or you want to shoot for a meeting on Thursday instead?

**Comment by JaredKramer40** [comment]
Tomorrow before class should work.  I don't think it's a big issue.  What would the host be when connecting? @RJP43

**Comment by RJP43** [comment]
@JaredKramer40 you will have to grab all the info about logging on through a FTP client from the announcement on Courseweb. We can't share the specific connection info here on GitHub since this is a public environment and the access info is private. I couldn't even find your folder on the PSC, but if you can complete or have completed the step of creating your password  through the PSC THEN I for sure can help you get it all set up tomorrow before class.

**Comment by RJP43** [comment]
@JaredKramer40 Plan to meet in the classroom around 9:30-9:45ish?

**Comment by JaredKramer40** [comment]
@RJP43  Okay sounds good! I will try to create my password through the PSC tonight. Thanks!

**Comment by ebeshero** [comment]
@JaredKramer40 @RJP43 I'll try to get there early, but I bet you'll figure this out before class since it sounds like you just need to do the first-time setup of the FTP client. You're on a mac, right?

**Comment by JaredKramer40** [comment]
@ebeshero @RJP43   Yes I am on a mac. And I just set up my password and am on Fetch now. So yeah I think I will be able to figure it out before class.

**Comment by ebeshero** [comment]
@JaredKramer40 Leave a message here once you've uploaded stuff to newtFire (or if you've got a question)--maybe you've got this. 👍

**Comment by RJP43** [comment]
Great @JaredKramer40! Definitely go ahead with moving your files over to your PSC webspace and we will still meet before class tomorrow to verify everything is up and running ok.

**Comment by JaredKramer40** [comment]
Okay awesome! Thanks! @RJP43 @ebeshero

**Comment by RJP43** [comment]
@JaredKramer40 SO there wasn't much improvement on your site from HTML Exercise 1 to HTML Exercise 2; however, you do have some working javaScript on your site and now with all your added experience from creating theGraveyard site I think you can really clean this site up. Consider using this web space as a professional portfolio for your digital scholarship. Remember, if you are considering the digital studies certificate a digital portfolio is a requirement. I don't think there is much work to do here to turn this into a really great space to show off all your newfound digital skills.
